src/guidescanpy/flask/templates/job_query.html

```diff
@@ -97,11 +97,11 @@ <h5 class="card-header">{{ region }}</h5>
                                         <tbody>
                                         {% for value in values['hits'] %}
                                             <tr>
-                                                <td>{{ value['coordinate'] }}</td>
+                                                <td><a href="#" id="coordinate">{{ value['coordinate'] }}</a></td>
```

Comment:
I think we should add the `class` attribute here instead of `id`, since its common to all rows (and then pick up the class in jquery instead of the `id`).